src/pyedb/dotnet/edb_core/edb_data/primitives_data.py

```diff
@@ -135,15 +135,15 @@ def layer_name(self):
         except AttributeError:  # pragma: no cover
             return None
 
-    @layer_name.setter
```

Comment:
You just removed the fact that property decorates with `.setter`. You should put back the removed `@layer_name.setter`